src/defaultConfig.ts

```diff
@@ -8,14 +8,15 @@ export const defaultSecurityConfig = (serverlUrl: string): ModuleOptions => ({
     crossOriginOpenerPolicy: 'same-origin',
     crossOriginEmbedderPolicy: 'require-corp',
     contentSecurityPolicy: {
-      'base-uri': ["'self'"],
+      'base-uri': ["'none'"],
       'font-src': ["'self'", 'https:', 'data:'],
       'form-action': ["'self'"],
       'frame-ancestors': ["'self'"],
       'img-src': ["'self'", 'data:'],
       'object-src': ["'none'"],
       'script-src-attr': ["'none'"],
       'style-src': ["'self'", 'https:', "'unsafe-inline'"],
+      'script-src': ["'self'", 'https:', "'unsafe-inline'", "'strict-dynamic'", "'nonce-{{nonce}}'"],
```

Comment:
issue: I tested the app with this setup and I get the following error. Could you please check it out?

![image](https://github.com/Baroshem/nuxt-security/assets/37120330/2a9fa7cc-5c57-44f3-94f9-b4cf7b0f8c98)

Reply:
Looks like the random nonce value is not set in the headers. They still have the placeholders.
Do you have `{ nonce: true }` ? It should be the default

Reply:
Ah interesting. I had a duplicate property for nonce (both true an false but in different places in the config) but TS didn't catch it.

I removed the duplicate one and it works correctly.

Thanks for checking that. Works like a charm. I will merge it right now :)

Reply:
Let me double check though, I believe you might have caught an edge case in dev mode

Reply:
@Baroshem Can you let me know what was your double config setup ?
I still find it weird that the random nonce was not in the headers